diff --git a/apps/srt-live-transmit.cpp b/apps/srt-live-transmit.cpp
index 7d320c3..6015957 100644
--- a/apps/srt-live-transmit.cpp
+++ b/apps/srt-live-transmit.cpp
@@ -177,6 +177,7 @@ int main( int argc, char** argv )
     {
         cerr << "Usage: " << argv[0] << " [options] <input-uri> <output-uri>\n";
         cerr << "\t-t:<timeout=0> - exit timer in seconds\n";
+        cerr << "\t-taoc - abort timeout on connect (default no)\n";
         cerr << "\t-c:<chunk=1316> - max size of data read in one step\n";
         cerr << "\t-b:<bandwidth> - set SRT bandwidth\n";
         cerr << "\t-r:<report-frequency=0> - bandwidth report frequency\n";
@@ -190,6 +191,7 @@ int main( int argc, char** argv )
     }
 
     int timeout = stoi(Option("0", "t", "to", "timeout"), 0, 0);
+    bool timeout_abrt_conn = Option("no", "taoc", "timeout-abort-on-connect") != "no";
     unsigned long chunk = stoul(Option("0", "c", "chunk"), 0, 0);
     if ( chunk == 0 )
     {
@@ -469,6 +471,12 @@ int main( int argc, char** argv )
                                         <<  " connection"
                                         << endl;
                                 }
+                                if (timeout_abrt_conn && timeout > 0)
+                                {
+                                    if (!quiet)
+				        cerr << "TIMEOUT: cancel\n";
+                                    alarm(0);
+                                }
                                 if (issource)
                                     srcConnected = true;
                                 else
